vespa/test_integration_docker.py

```diff
@@ -1030,91 +985,63 @@ def bert_model_input_and_output(
 
     def test_instantiate_vespa_docker_from_container_name_or_id(self):
-        self.create_vespa_docker_from_container_name_or_id(
-            application_package=self.app_package, disk_folder=self.disk_folder
-        )
+        self.create_vespa_docker_from_container_name_or_id(application_package=self.app_package)
 
     @pytest.mark.skip(reason="Works locally but fails on Screwdriver")
     def test_redeploy_with_container_stopped(self):
-        self.redeploy_with_container_stopped(
-            application_package=self.app_package, disk_folder=self.disk_folder
-        )
+        self.redeploy_with_container_stopped(application_package=self.app_package)
 
     def test_redeploy_with_application_package_changes(self):
-        self.redeploy_with_application_package_changes(
-            application_package=self.app_package, disk_folder=self.disk_folder
-        )
+        self.redeploy_with_application_package_changes(application_package=self.app_package)
 
     def test_trigger_start_stop_and_restart_services(self):
-        self.trigger_start_stop_and_restart_services(
-            application_package=self.app_package, disk_folder=self.disk_folder
-        )
+        self.trigger_start_stop_and_restart_services(application_package=self.app_package)
 
     def tearDown(self) -> None:
-        shutil.rmtree(self.disk_folder, ignore_errors=True)
         self.vespa_docker.container.stop()
         self.vespa_docker.container.remove()
 
 
 class TestCord19DockerDeployment(TestDockerCommon):
     def setUp(self) -> None:
         self.app_package = create_cord19_application_package()
-        self.disk_folder = os.path.join(os.getenv("WORK_DIR"), "sample_application")
 
     def test_deploy(self):
-        self.deploy(application_package=self.app_package, disk_folder=self.disk_folder)
+        self.deploy(application_package=self.app_package)
 
     def tearDown(self) -> None:
-        shutil.rmtree(self.disk_folder, ignore_errors=True)
         self.vespa_docker.container.stop()
         self.vespa_docker.container.remove()
 
 
 class TestQaDockerDeployment(TestDockerCommon):
     def setUp(self) -> None:
         self.app_package = create_qa_application_package()
-        self.disk_folder = os.path.join(os.getenv("WORK_DIR"), "sample_application")
 
     def test_deploy(self):
-        self.deploy(application_package=self.app_package, disk_folder=self.disk_folder)
+        self.deploy(application_package=self.app_package)
         self.vespa_docker.container.stop()
         self.vespa_docker.container.remove()
 
     def test_deploy_image(self):
         self.deploy(application_package=self.app_package,
-                    disk_folder=self.disk_folder,
                     container_image="vespaengine/vespa:7.566.21")
         self.vespa_docker.container.stop()
         self.vespa_docker.container.remove()
 
     def tearDown(self) -> None:
-        shutil.rmtree(self.disk_folder, ignore_errors=True)
+        pass
```

Comment:
Should we remove the tearDown method if it is not doing anything?

Reply:
I considered that, I kept if for symmetry with the others, can remove

Reply:
removed